keycloak.tf

```diff
@@ -16,31 +16,48 @@ module "keycloak" {
   lb_zone_id   = module.core.lb_public_zone_id
 
   # app specific variables
-  name     = "shared-keycloak"
+  name     = "keycloak"
   dns_name = local.keycloak_dns_name
   image    = var.image_shared_keycloak
   cpu      = "2048"
   memory   = "4096"
+
+  healthcheck                       = "/admin/"
```

Comment:
i thought this path was supposed to be /health

https://www.keycloak.org/server/health